images/dask/README.md

```diff
@@ -1,13 +1,25 @@
-# dask image
+# dask Image
 
-This image exists to add necessary dependencies for dask schedulers
+This image exists to add necessary dependencies for Dask schedulers
 and workers, such as the Prometheus client and Python libraries
-used by analytics code. See [the dask docs](https://docs.dask.org/en/stable/how-to/manage-environments.html)
+used by analytics code. See [the Dask docs](https://docs.dask.org/en/stable/how-to/manage-environments.html)
 for some additional detail.
 
-## Building and pushing manually
+## Testing Changes
+
+A person with Docker set up locally can build a new version of the Dask image at any time after making changes.
+
+```bash
+docker build -t ghcr.io/cal-itp/data-infra/dask:2022.10.13 .
```

Comment:
What version number do you use here when testing local changes?

Reply:
This is a totally arbitrary decision, since it's not pushed at this stage, so in [696ac34](https://github.com/cal-itp/data-infra/pull/3156/commits/696ac3432bae8a47626ed6bf35dc191c0254d2b9) I replaced it with the same generic `[NEW VERSION NUMBER]` reference as elsewhere.